### ISSUE TITLE
Split `scraper_utils.py`, `legistar_utils.py`

### ISSUE THREAD

**Issue by dphoria** [CouncilDataProject/cdp-scrapers#95]
### Feature Description

Take out top-level utility functions out of `scraper_utils.py` and `legistar_utils.py` into files under `cdp_scrapers/utils`. The files have become too large. i.e. Those files will contain implementations for `IngestionModelScraper` and `LegistarScraper` only.

### Solution

Restructure so we will do something like

```Python
from cdp_scrapers.utils import reduced_list
from cdp_scrapers.utils.legistar import get_legistar_body
```

### Alternatives

Functionally, the current state is completely adequate. I think the change will make for more manageable code.

**Comment by dphoria** [comment]
`LegistarScraper.find_time_zone()` should be a base `utils` member. i.e.

```Python
from cdp_scrapers.utils import find_time_zone
```

**Comment by mdahwireng** [comment]
How can one identify top-level utility functions?

**Comment by dphoria** [comment]
Hello @mdahwireng . Thank you so much for your interest. That is a great question and I will add some more definition to this issue later today.

**Comment by dphoria** [comment]
As far as I can tell these are the functions defined at the file root level in those 2 files.

```
get_legistar_body
get_legistar_person
get_legistar_events_for_timespan
get_legistar_content_uris

reduced_list
str_simplified
parse_static_person
parse_static_file
sanitize_roles
```

**Comment by mdahwireng** [comment]
@dphoria Thanks. I will have a look at it

**Comment by mdahwireng** [comment]
> `LegistarScraper.find_time_zone()` should be a base `utils` member. i.e.
> 
> ```python
> from cdp_scrapers.utils import find_time_zone
> ```

@dphoria  `LegistarScraper` does not have a `find_time_zone()` method. Is there something I am missing?

**Comment by dphoria** [comment]
No, not at all. You are correct; the code has changed since when this issue was created. Sorry about that confusion.
